### ISSUE TITLE
Surface number of crbugs.com filed against a feature on list

### ISSUE THREAD

**Issue by ebidel** [GoogleChrome/chromium-dashboard#375]
Would be cool to show the # of related open bugs for a given feature. It would be an interesting way for web devs to judge if they want to use a feature or not.

**Comment by RByers** [comment]
I'm not sure that's really possible to do with the data we have.  We know the labels for the launch bug, but usually the labels aren't granular enough (eg. tons of features go under the Blink>Input label).  We could try looking at how many bugs are blocked on the launch bug, but most people probably don't bother tying bugs back to the launch bug.

Longer term, I'd love to be able to link to the web-platform-test results for each feature on each browser.  Again that won't be easy (and won't always be that meaningful in terms of numbers) but is probably still of some value.

**Comment by ebidel** [comment]
Is there a live page that has this results for each browser? I'd love to
set something up if there isn't :)

**Comment by RByers** [comment]
Not really.  There's [this](https://github.com/w3c/test-results) but it's manual and a subset. @jgraham and @foolip are starting to collect and study some of the results data (platform-predictability has a Q3 OKR to at least review the failures in Chrome that pass in all other browsers).  There's [debate](https://twitter.com/RickByers/status/751257426322595840) whether 'gamifying' the results is a good idea or not - we plan to discuss this more at TPAC.

**Comment by jgraham** [comment]
Since I don't Twitter, I will take this opportunity to note that I don't think "gamifying" the results is a good idea; theoretically it doesn't work well because not all tests are created equal and it's much more important to fix one bug that is breaking sites than 100 edge cases that no one has run into yet. In practice we have seen it creating perverse incentives with people reluctant to release tests that they don't pass, or making shallow fixes to score better (e.g. stubbing out APIs just to get points for the interface existing). As a practical point I don't want Mozilla's marketing to take an interest in the testing effort in terms of what we contribute or what we fix; it's already enough work to make people write cross-browser tests rather than vendor-specific tests.

On the other hand, I am in favour of making information about who is passing which tests available to browser vendors since I believe that if there isn't a score attached they will make good, responsible, use of it to target the most important bugs and build a better platform. As such I'm happy to help with the effort to make these results more visible; I certainly feel we are not using the web platform tests to their full potential at Mozilla because although we use them to prevent regressions, we don't often notice where they are failing, or know which failures are browser bugs as opposed to test bugs.

**Comment by RByers** [comment]
Thanks for your insight @jgraham.  I definitely defer to your expertise here (for now <grin>).  There's tons we can do to better leverage WPT before we worry about further motivating investments.  My primary interest here is having the data/tooling to help me motivate blink teams to ramp up their investments in the high-value places.  Once I feel that's been successful I may have larger ambitions <grin>.
